packages/typescript-estree/src/create-program/createWatchProgram.ts

```diff
@@ -355,7 +355,7 @@ function maybeInvalidateProgram(
    */
   let updatedProgram = existingWatch.getProgram().getProgram();
 
-  // In case this change causes problems in larger real world codebases
+  // In case this change causes problems in larger real world code bases
```

Comment:
I think in the programming world, "codebase" and "codebases" are both acceptable usages even if they are not commonly used outside of programming. I think it would be okay to add those as valid words.

Reply:
actually i already added it to allowed words :>